### ISSUE TITLE
Broken indent on create field in inner class

### ISSUE THREAD

**Issue by tivervac** [eclipse/xtext-xtend#134]
When inserting a field through the quickfix, in an inner class, the wrong indent is used and existing indent is broken.
```
class Foo {
	private static class Bar {

		new() {
			this.foobar = 0
		}
	}
}
```

Creating a field for foobar through the quickfix yields:
```
class Foo {
	private static class Bar {

	int foobar
	
	new() {
			this.foobar = 0
		}
	}
}
```

**Comment by cdietrich** [comment]
all quickfixes are affected by this issues cause they use hard coded indentation levels

**Comment by tivervac** [comment]
@cdietrich as your branch has been merged, can this be closed now?

**Comment by cdietrich** [comment]
yes thx for the hint: fixed in Xtext/Xtend 2.13
